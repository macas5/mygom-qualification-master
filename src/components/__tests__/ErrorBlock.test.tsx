import React from "react";
import renderer from 'react-test-renderer';
import ErrorBlock from "../ErrorBlock";

it('renders correctly with variable', () => {
  const tree = renderer
    .create(<ErrorBlock error="Error"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly without variable', () => {
  const tree = renderer
    .create(<ErrorBlock error=""/>).toJSON();
  expect(tree).toMatchSnapshot();
});