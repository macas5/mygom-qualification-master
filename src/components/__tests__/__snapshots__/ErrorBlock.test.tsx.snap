// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with variable 1`] = `
<div>
  Error
</div>
`;

exports[`renders correctly without variable 1`] = `null`;
